run-tests.php

```diff
@@ -434,7 +434,7 @@ function main(): void
                     break;
                 case 'r':
                 case 'l':
-                    $test_list = file($argv[++$i]);
+                    $test_list = [$argv[++$i]];
```

Comment:
I don't understand. The foreach is supposed to iterate over the lines from the file, which are the tests that failed in the last run. This will simply iterate once, over the name of the list file.

Reply:
I understand now. The input file should contain the list of the files to test and not the testfiles themselves. My bad, I misunderstood the help instruction.
```
    -l <file>   Read the testfiles to be executed from <file>. After the test
                has finished all failed tests are written to the same <file>.
                If the list is empty and no further test is specified then
                all tests are executed (same as: -r <file> -w <file>).

    -r <file>   Read the testfiles to be executed from <file>.
```
@iluuu1994 Thanks for the clarification. I will close this pull request.

Reply:
Yes. If you want to specify a single file, you can simply add it to the inputs with no prefix.